.github/issue-labeler.yml

```diff
@@ -0,0 +1,10 @@
+# List of labels which should be assigned to issues based on a regex
+windows:
+  - 'Os\/Arch:\s*windows'
+
+macos:
+  - 'Os\/Arch:\s*darwin'
+
```

Comment:
For completeness, I'd love a `linux` label as well.

Reply:
The problem is that linux is always in the version output since the server always runs on linux. So it would label every issue linux. I do not see how that is useful so I didn't add one.

Reply:
I agree. It'll only make sense to mark as linux if it's neither macos nor windows.